importScripts/sampleRecords/additionalAdministration.destination.json

```diff
@@ -142,18 +174,18 @@
       "ca_objects.deaccession_type_id":"Item discarded",
       "ca_objects.LastEditDate":"16 February 2011",
       "ca_objects.LastEditBy":"VandB",
-      "ca_storage_locations":"Costume Room Box 225"
+      "usualLocationHierarchy":" (id:1, ) => Costume Room Box 225 (id:13, room)"
```

Comment:
This comes from the data that they haven't cleaned - the data stored in the Objects table are different from the values in the LocationHomes table which was what the "controlled" vocabulary of storage locations. I gave them the LocationHomes table for cleaning, and after loading that found that a number of records (including this one) no longer had a storage location.